aten/src/ATen/native/vulkan/Vulkan.cpp

```diff
@@ -896,6 +908,79 @@ VBuffer makeUniformConstBuffer(void* ptr, VkDeviceSize size) {
   return constBuffer;
 }
 
+ComputeUnitFactory::ComputeUnitFactory(const VkDevice& device)
```

Comment:
Typically opaque handles in Vulkan like VkDevice etc. are just a typedef to an integer or a pointer so there's no need to pass them by reference.